lms/djangoapps/courseware/courses.py

```diff
@@ -459,26 +458,9 @@ def get_courses(user, org=None, filter_=None):
     Returns a list of courses available, sorted by course.number and optionally
     filtered by org code (case-insensitive).
     """
-    # In the event we don't want any course tiles displayed
```

Comment:
There's still other fork in this file, but this part is clean now (just the 3 lines added below):

```python
courses = get_visible_courses(org=org, filter_=filter_)
if courses is not None:
    return courses
```